LongitudinalDifferenceInDifferencesPy/didDesigner.py

```diff
@@ -176,6 +176,10 @@ def estimateSummmary(self,df):
         print("Area under the Curve Estimate:")
         print(self.AreaUnderCurveEstimate(df))
 
+        print("")
```

Comment:
Nice. I like this better than running the function individually. Ca you edit the wiki?

Reply:
I made a minor edit here, but same behavior